submissions/solaryasha/html-css-popup/css/styles.css

```diff
@@ -1,12 +1,14 @@
+* { 
+    box-sizing: border-box;
+} 
 body {
     font-size: 15px;
     font-family: 'Segoe UI', 'Roboto', arial, sans-serif;
-    overflow: hidden;
+    padding: 0;
+    margin: 0;
+    outline: none;
```

Comment:
Be careful with the disabling outline for the whole page. Because you turn off default style on `:focus` state. Ьake sure you add styles instead of disabled styles.